Makefile

```
@@ -247,6 +254,7 @@ test-preview: ## Generate non-destructive Pulumi previews for configured stacks.
 test-preview-unprivileged: ## Generate an unprivileged placeholder preview artifact.
```

checkmake (0.3.2)

[warning] Target body for "test-preview-unprivileged" exceeds allowed length of 5 lines (6).

(maxbodylength)